apis/Google.Cloud.Spanner.Admin.Instance.V1/Google.Cloud.Spanner.Admin.Instance.V1/Google.Cloud.Spanner.Admin.Instance.V1.csproj

```diff
@@ -21,12 +21,11 @@
     <PackageProjectUrl>https://github.com/GoogleCloudPlatform/google-cloud-dotnet</PackageProjectUrl>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/GoogleCloudPlatform/google-cloud-dotnet</RepositoryUrl>
+    <CodeAnalysisRuleSet>..\..\..\grpc.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="ConfigureAwaitChecker.Analyzer" Version="1.0.0-beta4" PrivateAssets="All" />
     <PackageReference Include="Google.Api.Gax.Grpc" Version="2.1.0-beta02" />
-    <PackageReference Include="Google.Cloud.Iam.V1" Version="1.0.0-beta12" />
```

Comment:
Looks like this was a merge conflict resolution issue.  this should have stayed around

Reply:
I don't think it was a conflict. See my comment on #1455. Re-running the project generator automatically removed these.